frontend/resources/translations/fr-FR.json

```diff
@@ -237,6 +242,11 @@
     "helpCenter": "Centre d'aide",
     "add": "Ajouter",
     "yes": "Oui",
+    "Done": "Fait",
+    "Cancel": "Annuler",
```

Comment:
Thanks for contributing! These two tokens are no longer being used, but it seems like they are still appearing on the inlang editor. I'm submitting a patch to remove them now. Could you kindly refresh your PR after that has been merged?

Reply:
Okay I've merged it. Please remove these two keys